site_scons/site_tools/protoc/__init__.py

```diff
@@ -81,17 +79,16 @@ def _detect(env):
     else:
         if not protoc_found:
             raise SCons.Errors.StopError(ProtocCompilerNotFound,
-                "Could not detect protoc compiler")
+                                         "Could not detect protoc compiler")
         if not protoc_gen_go_found:
             raise SCons.Errors.StopError(GoProtocCompilerNotFound,
-                "Could not detect protoc-gen-go")
+                                         "Could not detect protoc-gen-go")
         if not grpc_tools_found:
             raise SCons.Errors.StopError(PythonGRPCCompilerNotFound,
-                "grpc_tools.protoc python module is not installed")
+                                         "grpc_tools.protoc python module is not installed")
```

Comment:
(pylint-line-too-long) Line too long (92/80)

Reply:
this tool isn't used, I think we could just remove it.

Reply:
Which tool?  I spent quite some time getting this to work and remove a bunch of issues plus it looks like it's python2 only so I'd like to remove what we can from here.